unittest

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env sh
 
+# Note: use preferably `tox` to run all tests from an isolated environment
+
 python -m unittest discover -s test -p '*_test.py'
```

Comment:
These `unittest*` files can now be deleted (`tox` works the same on OSX, linux, windows...)